packages/constants/tests/php/test-constants.php

```diff
@@ -156,7 +155,7 @@ public function test_jetpack_constants_get_constant_use_filter_value() {
 		$apply_filters_spy = new Spy(
 			'Automattic\Jetpack',
 			'apply_filters',
-			function ( $filter_name, $value, $name ) use ( $test_constant_value ) {
+			function ( $filter_name, $value, $name ) use ( $test_constant_value ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
Could just remove the unused variables here, if you want.

I wouldn't recommend that for actual methods intended for being passed as callbacks, but this is just a closure.